### ISSUE TITLE
[Bug] Homescreen is bugged

### ISSUE THREAD

**Issue by MaartenSomay** [YTLitePlus/YTLitePlus#116]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Have you read the FAQ?

- [X] Yes, I read the FAQ

### Device info

- iOS/iPadOS version: 17.4.1
- Device model: Iphone 14 pro max
- Sideload tool (AltStore, Sideloadly, TrollStore,...): Esign
- The specific version of YTLitePlus (**latest** or **newest** is **NOT** a version number!): 19.12.3-3.0.1

### Describe the bug

Homescreen is bugged when ‘All’ is selected. Worked just fine a couple hours ago.
![IMG_9220](https://github.com/YTLitePlus/YTLitePlus/assets/37876797/90c47b6b-5a84-4740-9ba7-b4822593039f)

### Steps to reproduce the issue

1.  Open youtube
2. 
3. 

### Are you using the newest version of YTLitePlus? If not, why?

✅ Yes, I'm using the latest version of YTLitePlus right now

### Does the issue happen with the official YouTube from AppStore?

❌ No, It doesn't

**Comment by ErickGuerrero** [comment]
Same problem here. Seems to be a problem with Adblock.

**Comment by MaartenSomay** [comment]
> Same problem here. Seems to be an error with Adblock.

Correct. Turning off ‘remove ads’ fixes this. But now I have ads :(

**Comment by cooldude-91** [comment]
Same issue here

**Comment by spookwav** [comment]
Same here. Issue doesn't go away when i disable remove ads.

**Comment by MaartenSomay** [comment]
@Dev 

any ETA when its fixed? This bug is really annoying. Thanks

**Comment by haomuch** [comment]
same issue for me. Tuned off remove ads can fix it. Wait for the ads module update.

**Comment by hellosquishies** [comment]
It’s an interesting issue, I had the same problem for weeks now but yesterday it suddenly disappeared and it’s all normal again

**Comment by haomuch** [comment]
From the uYouplus enhanced project, which also finds this bug, spoofing to version 18.16.2 could fix it. But spoofing doesn’t work in YTLite Plus. Could the developer take a look at it?

**Comment by binnichtaktiv** [comment]
It’s even worse now😭 Latest yt version…
![RPReplay_Final1714729696](https://github.com/YTLitePlus/YTLitePlus/assets/96953964/a5a15d13-538f-49ab-b022-cfbb43f24434)

**Comment by Balackburn** [comment]
@binnichtaktiv I am sorry their isn't any fix for now, waiting for a YTLite update

**Comment by shahin365** [comment]
> @binnichtaktiv I am sorry their isn't any fix for now, waiting for a YTLite update

I just wanted to inform you that this https://github.com/dayanch96/YTLite/issues/71 is incorrect. I have tried disabling the 'Hide Shorts' option to fix the blank space problem, but it didn't work in my case. Some users on GitHub & Reddit have also reported facing the same issue in the past few weeks and disabling the 'Hide Shorts' didn't solve it. It seems that the only solution that works for everyone is to disable the ad-blocker, but that defeats the purpose of using YTLite. The root cause of this bug is the ad-blocker, not the 'Hide Shorts' feature.

**Comment by Balackburn** [comment]
> > @binnichtaktiv I am sorry their isn't any fix for now, waiting for a YTLite update
> 
> 
> 
> I just wanted to inform you that this https://github.com/dayanch96/YTLite/issues/71 is incorrect. I have tried disabling the 'Hide Shorts' option to fix the blank space problem, but it didn't work in my case. Some users on GitHub & Reddit have also reported facing the same issue in the past few weeks and disabling the 'Hide Shorts' didn't solve it. It seems that the only solution that works for everyone is to disable the ad-blocker, but that defeats the purpose of using YTLite. The root cause of this bug is the ad-blocker, not the 'Hide Shorts' feature.
> 
> 

The hiding short feature does put a blank space, this is a new bug, I don't have it myself but the two options put a blank space in the app. This is the biggest issue for now.

**Comment by binnichtaktiv** [comment]
> > > @binnichtaktiv I am sorry their isn't any fix for now, waiting for a YTLite update
> > 
> > 
> > I just wanted to inform you that this [dayanch96/YTLite#71](https://github.com/dayanch96/YTLite/issues/71) is incorrect. I have tried disabling the 'Hide Shorts' option to fix the blank space problem, but it didn't work in my case. Some users on GitHub & Reddit have also reported facing the same issue in the past few weeks and disabling the 'Hide Shorts' didn't solve it. It seems that the only solution that works for everyone is to disable the ad-blocker, but that defeats the purpose of using YTLite. The root cause of this bug is the ad-blocker, not the 'Hide Shorts' feature.
> 
> The hiding short feature does put a blank space, this is a new bug, I don't have it myself but the two options put a blank space in the app. This is the biggest issue for now.

exactly.. I dont have hide shorts enabled and I still have this bug.. Lets hope we get a fix soon🥲

**Comment by nbaytekin18** [comment]
I installed v19.19.5-3.0.1 two days ago. It was buggy when I first installed it, got fixed yesterday, but today it is back again.

**Comment by samuelc02** [comment]
This is so weird. Why does it only affect the "all" tab but not other tabs? And somehow scrolling down a screen and a half the videos reappear? Even though it's still bit buggy

**Comment by a-unknown-user** [comment]
Yep. Either have no AdBlock or have to scroll through a buggy homepage, let's hope it gets fixed in a future release.

**Comment by binnichtaktiv** [comment]
@Balackburn wirst du YTLite updaten? YtPlus Version 4.0 came out yesterday. I don’t think he will open source it. But can you still update? You can just inject it and not build it in github actions. I like this repo because it has all tweaks I need 🫶🏻
https://t.me/dvntms/58

**Comment by binnichtaktiv** [comment]
@Balackburn will you update YtLitePlus? YTPlus version 4.0 was released yesterday but I don't think it will be open source. many bugs have been fixed and there are new features. I like this repo because it has all the tweaks I need🫶🏻

https://t.me/dvntms/58

**Comment by Balackburn** [comment]
@binnichtaktiv Yes I am working on it right now

**Comment by binnichtaktiv** [comment]
> @binnichtaktiv Yes I am working on it right now

Thank you so much😻

**Comment by binnichtaktiv** [comment]
@Balackburn sry for @ but YTPlus v4.0.1 released can you pls update?

**Comment by David735453** [comment]
> @binnichtaktiv Yes I am working on it right now

Any Updates?

**Comment by David735453** [comment]
> @David735453 It should be fixed with latest version

It's working thanks for your Time to give us a wonderful YouTube Experience.
